test/apicoverage-webhook/display_rules.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/knative/test-infra/tools/webhook-apicoverage/coveragecalculator"
+)
+
+// display_rules.go contains all the display rules specified by knative serving to display json type like result display.
+
+// PackageDisplayRule rule specifies how package name needs to be displayed for json type like result display
+func PackageDisplayRule(packageName string) string {
```

Comment:
Same comment about these seeming to be generic enough to warrant their place in a generic collection.

Reply:
Again, some other repo might want to display it in a way they see fit. Generally speaking, I want the repos to have the ability to define the way they want to use tool rather than imposing anything. If we see a pattern across the repos then in my mind it makes sense to re-factor and provide generic rules.